INSTALLER-HACKING.md

```diff
@@ -0,0 +1,63 @@
+# Running local builds with the installer
+
+If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you.
+
+## Step 0: Prepare your credentials
+
+You will need to do this once, and every time your token resets.
+
+1. Get your CI credentials by logging on to the [CI server](https://api.ci.openshift.org/console) using your GitHub account and clicking _Copy Login Command_ from the menu on the top right.
+
+2. Execute that command, which should look like
+
+```
+oc login https://api.ci.openshift.org --token=XXXX
+```
+
+3. Copy that token, and use it to log in to the registry with podman:
+
+```
+podman login -u <YOUR GITHUB ID> -p <YOUR TOKEN> registry.svc.ci.openshift.org
+```
+
+Tokens expire regularly - about every week.
+
+
+## Step 1: Prepare the cluster & environment
+
+You will need to do this for every new cluster
+
+1. In one window, prepare the cluster
+```
+export CLUSTER_DIR=<your cluster directory>
+openshift-install --dir=$CLUSTER_DIR create manifests
+```
+
+2. In another window, prepare the operator. This will apply some overrides that disable the default cluster-network-operator. It will also extract the release image
+
+```
+export CLUSTER_DIR=<your cluster directory>
+hack/run-locally.sh prepare
+```
+
+3. Optional: Override images
+If you are doing downstream image development, e.g. working on openshift-sdn or the like, you should build and publish a custom image now. Then, edit `env.sh` to point to your development image
+
+## Step 2: Create the cluster
+
+1. In the installer window, 
+
+```
+openshift-install --dir=$CLUSTER_DIR create cluster
+```
```

Comment:
... and don't wait for it to finish; the install can't complete until you bring up the network operator, right?